BranchMetrics/cpp-branch-deep-linking-attribution
# SDK-2041 Windows Stop calling v1 profile and v1 logout

## Reference
SDK-2041 -- [Windows] Stop calling /v1/profile and /v1/logout.
https://branch.atlassian.net/browse/SDK-2041

## Summary
Store and clear the identity locally without making requests to the endpoints.

## Type Of Change


## Testing Instructions

->Some files have been removed and package versions are updated. So please check code compilation is successful.
-> Following scenarios should work-
 * User should be able to set identity and then retrieve it later.
 * User should get back following  params via callback for setIdentity and logout.
        -- randomized_bundle_token
        -- session_id
        -- link
* User should be able to logout. On logout only identity is cleared.
* SetIdentity and logout should not work if tracking is disabled. 
-> Ensure No network calls are made for endpoints /v1/profile and v1/logout(check in logs).









cc @BranchMetrics/saas-sdk-devs for visibility.
